Assets/Scripts/KeyBindings/KeyBindings.cs

```diff
@@ -43,4 +46,13 @@ public static bool SetKeyBinding(string key, KeyCode binding)
             return false;
         }
     }
+    public static Dictionary<String, KeyCode> Get()
+    {
+        return bindings;
```

Comment:
Name of function is not explanatory enough

Reply:
well it gets the value of the per-se datatype, do you have any better names?